tools/prepare-source.sh

```diff
@@ -13,11 +13,11 @@ $GO_BINARY download
 which goimports || $GO_BINARY install golang.org/x/tools/cmd/goimports@latest
 
 # Ensure that all code has been regenerated from its sources
-$GO_BINARY generate ./...
+$GO_BINARY generate -mod=mod ./...
 
 # ... the code is formatted correctly, ...
 goimports -w internal cmd
-$GO_BINARY fmt ./internal/... ./cmd/...
+$GO_BINARY fmt -mod=mod ./internal/... ./cmd/...
 
 # ... and that go.mod and go.sum are up to date.
 $GO_BINARY mod tidy
```

Comment:
I was getting errors when generating and formatting code because oapi-codegen is actually not imported at all therefore it is also not vendored. I am not sure how this previously worked, I believe this was due to cache. Once the command is installed it works.

The workaround is to use non-vendored modules for the generation and formatting. Could not figure out a better solution, alternatively we could import the generator via `import _ "..."` but that would pull unwanted dependencies into one of the binaries which is not what I would like to do.